lib/IDE/CodeCompletion.cpp

```diff
@@ -782,12 +777,20 @@ static void addDeclKeywords(CodeCompletionResultSink &Sink, DeclContext *DC,
         DeclAttribute::isConcurrencyOnly(*DAK))
       return;
 
-    addKeyword(Sink, Name, Kind, /*TypeAnnotation=*/"", getFlair(Kind, DAK));
+    CodeCompletionFlair flair = getFlair(Kind, DAK);
+
+    // Special case for 'actor'. Get the same flair with 'kw_class'.
+    if (Kind == CodeCompletionKeywordKind::None && Name == "actor")
+      flair = getFlair(CodeCompletionKeywordKind::kw_class, None);
+
+    addKeyword(Sink, Name, Kind, /*TypeAnnotation=*/"", flair);
   };
 
 #define DECL_KEYWORD(kw)                                                       \
   AddDeclKeyword(#kw, CodeCompletionKeywordKind::kw_##kw, None);
 #include "swift/Syntax/TokenKinds.def"
+  // Manually add "actor" because it's a contextual keyword.
+  AddDeclKeyword("actor", CodeCompletionKeywordKind::None, None);
```

Comment:
Wouldn’t it be simpler to remove the special case in `AddDeclKeyword` and change this
```suggestion
  AddDeclKeyword("actor", CodeCompletionKeywordKind::kw_class, None);
```

Reply:
Isn’t that exactly what you’re doing in line 783-784? Or am I missing something?

Reply:
No, I don't rewrite the `Kind`, but I just use `kw_class` _to get the flair_ for `actor`.
Using `kw_class` results
```
Keyword[class]/None: actor
```
I admit CodeCompletionKeywordKind is not used in many place, but for example, `CodeCompletion::LimitedResultView` has an ability to filter the results by the kind.

Reply:
Ah, I see. I that case forget what I said.